main_server.py

```
@@ -3364,11 +3366,18 @@ async def scan_local_workshop_items(request: Request):
         published_items = []
         item_id = 1
         
+        # 获取Steam下载的workshop路径，这个路径需要被排除
```

Ruff (0.14.6)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)